TopMovies/View/CategoryVC/MoviesCategoryVC.swift

```diff
@@ -0,0 +1,157 @@
+//
+//  MoviesCategoryVC.swift
+//  TopMovies
+//
+//  Created by Macbook Pro  on 17.02.2021.
+//
+
+import UIKit
+import SnapKit
+
+// MARK: - Props struct
+struct MoviesCategoryVCProps {
+  let categoryId: MovieCategory.ID
+  let categoryName: String
+  let loadProgress: LoadProgress
+  var moviesState: MoviesState
```

Comment:
`props` shouldn't contain variables, only constants that represent current state.
Otherwise, it allows you to copy props and change something inside of them.